Life-Expectancy-and-GDP-Starter/life_expectancy_gdp.ipynb

```diff
@@ -138,8 +162,11 @@
    "source": [
     "* The data contains 96 entries with no null values\n",
     "* The data types are correct as the `Country` variable is a string while the `Life expectancy at birth (years)` and `GDP` are float. The `Year` variable is an integer\n",
+    "* The countries present in the data are: Chile, China, Germany, Mexico, United States of America and Zimbabwe.\n",
```

Comment:
We need a line saying that we renamed data in the dataset.
"* The country `United States of America` was renamed to `USA`."

Reply:
Correct!